.github/workflows/build_for_deployment.yaml

```diff
@@ -0,0 +1,50 @@
+name: mara-app
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  mara-app-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6', '3.7', '3.7-dev']
+    steps:
+      - name: Chechout code
+        uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install and test application
+        env:
+          pythonversion: ${{ matrix.python-version }}
+        run: |
+          python -c "import sys; print(sys.version)"
+          pip install .[test]
+          echo Finished successful build with Python $pythonversion
+  mara-app-deploy:
+    needs: mara-app-build
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Chechout code
+        uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install wheel and build the package
+        run: |
+          git clean -xfd
+          python -m pip install wheel
+          python setup.py sdist bdist_wheel
+      - name: Publish package
+        uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          user: __token__
+          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
```

Comment:
TEST?

Reply:
I was using https://test.pypi.org/ to test the publishing. In case you agree on the PR, we can change it directly here to PYPI_API_TOKEN which should be provided by someone with the access to PYPI of Mara-App package and saved to the secrets.